src/targets/gpu/lowering.cpp

```diff
@@ -251,7 +252,12 @@ struct miopen_apply
             assert(refs.size() == 2);
             auto output = insert_allocation(ins, ins->get_shape());
```

Comment:
You need to do another `insert_allocation` for the workspace as well.

Reply:
Made changes for allocating workspace in lowering.